src/ios/OpenTokPlugin.m

```diff
@@ -450,7 +465,9 @@ - (void)signal:(CDVInvokedUrlCommand*)command{
     OTConnection* c = [connectionDictionary objectForKey: signalRecipientId];
 
     NSLog(@"iOS signaling to connection %@", c);
-    [_session signalWithType:signalType string:signalPayload connection:c error:nil];
+    NSString* sessionId = [command.arguments objectAtIndex:3];
```

Comment:
move to the beginning of the function next to all the others `[command.arguments objectAtIndex`